pkg/config/config.go

```diff
@@ -1166,6 +1166,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("capture_lambda_payload", false)
 	config.BindEnvAndSetDefault("serverless.trace_enabled", false, "DD_TRACE_ENABLED")
 	config.BindEnvAndSetDefault("serverless.trace_managed_services", true, "DD_TRACE_MANAGED_SERVICES")
+	config.BindEnvAndSetDefault("service_mapping", nil, "DD_SERVICE_MAPPING")
```

Comment:
Any reason why we wouldn't have this scoped to `serverless.` prefix?

Reply:
honestly, no
I just used the above lines as a guide to get this working with the cfg pkg and just left it there. Prob better to add the prefix for now. Ty